.travis.yml

```diff
@@ -55,28 +55,44 @@ jobs:
       env:
         - RELEASES_FILE=FreeOrionSDK_*.tar.bz2
         - RELEASES_TAG_NAME=$TRAVIS_TAG
+        - HOMEBREW_NO_AUTO_UPDATE=1
       before_install:
         - brew install gpatch
       script:
         - cmake -G "Xcode" -D BOOST_PARALLEL_JOBS=$(sysctl hw.ncpu | awk '{print $2}') ..
-        - travis_wait 70 cmake --build . --config RelWithDebInfo -- -parallelizeTargets -jobs $(sysctl hw.ncpu | awk '{print $2}')
+        - travis_wait 49 cmake --build . --config RelWithDebInfo -- -parallelizeTargets -jobs $(sysctl hw.ncpu | awk '{print $2}')
 
     - name: Build FreeOrionSDK for Windows (x86)
       <<: *_build_sdk_common
       os: windows
+      compiler: cl
       env:
         - RELEASES_FILE=FreeOrionSDK_*.zip
         - RELEASES_TAG_NAME=$TRAVIS_TAG
+        - CC=cl.exe
+        - CXX=cl.exe
+        - CC_FOR_BUILD=cl.exe
+        - CXX_FOR_BUILD=cl.exe
+      before_install:
+        - choco uninstall mingw
       script:
+
         - cmake -G "Visual Studio 15" -T v141 -A Win32 ..
         - travis_wait 70 cmake --build . --config RelWithDebInfo -- //maxcpucount //property:BuildInParallel=true //property:CL_MPCount=2 //verbosity:minimal
 
     - name: Build FreeOrionSDK for Windows (x64)
       <<: *_build_sdk_common
       os: windows
+      compiler: cl
       env:
         - RELEASES_FILE=FreeOrionSDK_*.zip
         - RELEASES_TAG_NAME=$TRAVIS_TAG
+        - CC=cl.exe
+        - CXX=cl.exe
+        - CC_FOR_BUILD=cl.exe
+        - CXX_FOR_BUILD=cl.exe
```

Comment:
Are these the relevant environment variables...?

Reply:
Travis will set them all to gcc/g++ if not set so I think it better to set all of them.